modules/metering-use-mysql-or-postgresql-for-hive.adoc

```diff
@@ -10,7 +10,7 @@ The default installation of metering configures Hive to use an embedded Java dat
 There are 4 configuration options you can use to control the database used by Hive metastore: url, driver, username, and password.
 
 
-You will create your MySQL or Postgres instance with a username and secret. Then either in the Kubernetes CLI or by making a yaml file, you will make a secret. The secretName you use for this secret needs to map to you Metering CR spec.hive.spec.config.db.secretName.
+You will create your MySQL or Postgres instance with a username and secret. Then either in the Kubernetes CLI or by making a yaml file, you will make a secret. The secretName you use for this secret needs to map to you Metering CR `spec.hive.spec.config.db.secretName`.
```

Comment:
```suggestion
You will create your MySQL or Postgres instance with a username and secret. Then either in the Kubernetes CLI or by making a yaml file, you will make a secret. The `secretName` you use for this secret needs to map to your Metering CR `spec.hive.spec.config.db.secretName`.
```
If backticks highlight direct literal elements in `.adoc` files, might as well be consistent.
```

Reply:
Should probably be referencing the OpenShift CLI here, not Kube.